### PR TITLE
ERC20：support approve operation

### LINKED ISSUE #29: show allowance and aproove button
image

### PR DESCRIPTION
**Pull request by Bigdragon123**
show allowance and aproove button

### CODE REVIEW

File: public/translations/zh_CN.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "address.copied": "The address is copied.",
   "invalid.input": "Invalid Input",
   "submit": "提交",
+  "approve": "认证",
```
Comment (uu-z) [inline public/translations/zh_CN.json]: 认证 -> 授权

File: src/pages/erc20.tsx [hunk 2]
```diff
@@ -22,16 +32,17 @@ const ERC20 = observer(() => {
     curToken: null as TokenState,
     isOpenTokenList: new BooleanState(),
     get loading() {
-      return store.curToken?.transfer.loading;
+      return store.curToken?.transfer.loading || store.curToken?.approve.loading;
```
Comment (uu-z) [inline src/pages/erc20.tsx]: we could remove loading and just use `transfer.loading` or `approve.loading` for each button

**Review by uu-z — CHANGES_REQUESTED** [review]

File: src/pages/erc20.tsx (hunk 2 above)
Comment (Bigdragon123) [reply src/pages/erc20.tsx]: already edited

**Comment by uu-z** [comment]
image
approve button shoud start loading when sending tx

**Comment by uu-z** [comment]
image
Submit -> Transfer

**Comment by Bigdragon123** [comment]
> image Submit -> Transfer

alreday edited

**Comment by Bigdragon123** [comment]
> image approve button shoud start loading when sending tx

already edited

**Pull request merged**